st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -92,6 +125,50 @@ def _get_default(_actionchain):
         return _actionchain.chain[0].name
 
     @staticmethod
+    def _get_all_nodes(action_chain):
```

Comment:
I follow an existing style approach, but honestly, I'm not a big fan of it.

I would rather move those utility functions on the actual `ActionChain` system model class. What do others think?